src/index.html

```diff
@@ -9,6 +9,10 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Stop watch</h1>
+    <div class="clock">
+      <div class="center-circle"></div>
+      <div class="hour-hand"></div>
+      <div class="minute-hand"></div>
```

Comment:
```suggestion
    <div class="clock">
      <div class="clock__center-circle"></div>
      <div class="clock__hour-hand"></div>
      <div class="clock__minute-hand"></div>
```
![image](https://github.com/mate-academy/layout_stop-watch/assets/96409563/6499385b-8f03-4157-b483-50071529c2ff)